components/time/src/zone/mod.rs

```diff
@@ -203,19 +224,29 @@ pub(crate) mod ule {
     #[cfg_attr(feature = "serde", derive(serde::Deserialize))]
     #[non_exhaustive]
     pub enum TimeZoneVariant {
-        /// The variant corresponding to `"standard"` in CLDR.
+        /// Indicates the period in a time zone with a lower UTC offset than `Daylight`.
+        ///
+        /// Every time zone supports the `Standard` variant.
         ///
-        /// The semantics vary from time zone to time zone. The time zone display
-        /// name of this variant may or may not be called "Standard Time".
+        /// ❗ Clients should generally NOT construct this variant directly. Instead, use:
+        /// * [`TimeZoneVariant::from_rearguard_isdst`]
+        /// * [`TimeZoneInfo::infer_variant`](crate::TimeZoneInfo::infer_variant)
         ///
-        /// This is the variant with the lower UTC offset.
+        /// Although CLDR calls this variant `"standard"`, the semantics vary
+        /// from time zone to time zone. The time zone display name of this variant
+        /// may or may not be called "Standard Time".
         Standard = 0,
-        /// The variant corresponding to `"daylight"` in CLDR.
+        /// Indicates the period in a time zone with a higher UTC offset than `Standard`.
+        ///
+        /// Not every time zone supports the `Daylight` variant.
         ///
-        /// The semantics vary from time zone to time zone. The time zone display
-        /// name of this variant may or may not be called "Daylight Time".
+        /// ❗ Clients should generally NOT construct this variant directly. Instead, use:
+        /// * [`TimeZoneVariant::from_rearguard_isdst`]
+        /// * [`TimeZoneInfo::infer_variant`](crate::TimeZoneInfo::infer_variant)
         ///
-        /// This is the variant with the higher UTC offset.
+        /// Although CLDR calls this variant `"daylight"`, the semantics vary
+        /// from time zone to time zone. The time zone display name of this variant
```

Comment:
Again, I don't actually think the "semantics" vary, it's always the higher-offset zone. The display names vary.

Reply:
Reworded